IFComp/lib/IFComp/Schema/Result/Entry.pm

```diff
@@ -1078,7 +1078,10 @@ body {
 <script type="text/javascript">
 game_options = {
   use_query_story: false,
-  set_page_title: true
+  set_page_title: true,
+  recording_url: '../../../play/$entry_id/transcribe',
```

Comment:
Is there a reason all of the URLs, including the static assets, are relative? Why not just '/play/...' or '/static/...'?